<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseAudioCodec</class>
 <widget class="QWidget" name="ChooseAudioCodec">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Audio Codec</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="audio_codec"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Bitrate, kb/s:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="audio_bitrate">
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Numbers of channels</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="num_chan"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="audio_adv">
     <property name="text">
      <string>Advanced (not yet implemented)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
